packages/core/src/components/Pagination/index.tsx

```diff
@@ -150,14 +154,16 @@ export class Pagination extends React.Component<Props & WithStylesProps> {
           <Text muted>
             {showBookends && pageCount ? (
               <T
-                phrase="Page %{pageNumber} of %{pageCount}"
+                phrase="%{pageLabel} %{pageNumber} of %{pageCount}"
```

Comment:
and since `pageLabel?:` is optional - it can have `undefined` as a value, so we need translate without the `pageLabel` variable. 

something like (pseudo code):

```js
let paginationText = showBookends && pageCount ? 
  phrase="%{pageNumber} of %{pageCount}"  : 
  page;

if (pageLabel) {
  paginationText = showBookends && pageCount ?
    phrase="%{pageLabel} %{pageNumber} of %{pageCount}" : 
    phrase="%{pageLabel} %{pageNumber}";
}
```

Reply:
or make pageLabel required (empty string and no undefined) `pageLabel: string;` in the prop defs and have no default prop value